integrations/examples/instance.md

```diff
@@ -0,0 +1,279 @@
+
+_Please take a look at the_ [_setup steps for Rust_](../rust/setup.md)_._
+{% endhint %}
+
+```bash
+cargo new instance
+cd instance
+```
+
+This should generate two important files for us, `Cargo.toml` and `src/main.rs`. The `Cargo.toml` is a file that describes your project and its dependencies. The `src/main.rs` is the entry point for your project, and contains the `fn main() { .. }` that is run when the project is executed.
+
+We then modify the `Cargo.toml` to add the Wasmer dependencies as shown below:
+
+{% code title="Cargo.toml" %}
+```rust
+[package]
+name = "instance"
+version = "0.1.0"
+authors = ["The Wasmer Engineering Team <engineering@wasmer.io>"]
+edition = "2018"
+
+[dependencies]
+# The Wasmer API
+wasmer = "1.0.0-alpha4"
+```
+{% endcode %}
+{% endtab %}
+
+{% tab title="Go" %}
+{% hint style="info" %}
+The final code for this example can be found on [GitHub](https://github.com/wasmerio/wasmer/blob/master/examples/instance.rs).
+
+_Please take a look at the_ _setup steps for Go._
+{% endhint %}
+
+```text
+mkdir wasmer-example-instance
+cd wasmer-example-instance
+go mod init github.com/$USER/wasmer-example-instance
+```
+{% endtab %}
+{% endtabs %}
+
+Now that we have everything set up, let's go ahead and try it out!
+
+### Loading the WASM module
+
+The first step will be to load the WASM module we want to use. This is done by having its contents loaded as bytes:
+
+{% tabs %}
+{% tab title="Rust" %}
+```rust
+let wasm_bytes = wat2wasm(br#"
+(module
+  (type $add_one_t (func (param i32) (result i32)))
+  (func $add_one_f (type $add_one_t) (param $value i32) (result i32)
+    local.get $value
+    i32.const 1
+    i32.add)
+  (export "add_one" (func $add_one_f)))
+"#)?;
+```
+
+{% hint style="info" %}
+Here we are using the text representation of the WASM module. Wasmer wants to have a binary representation of the module so we have to use `wat2wasm` to do the translation.
+{% endhint %}
+{% endtab %}
+
+{% tab title="Go" %}
+```go
+wasmBytes := []byte(`
+(module
+  (type $add_one_t (func (param i32) (result i32)))
+  (func $add_one_f (type $add_one_t) (param $value i32) (result i32)
+    local.get $value
+    i32.const 1
+    i32.add)
+  (export "add_one" (func $add_one_f)))
+`)
+```
+{% endtab %}
+
+{% tab title="C/C++" %}
+```c
+FILE* file = fopen("module.wasm", "rb");
+
+if (!file) {
+  printf("> Error loading module!\n");
+  
+  return 1;
+}
+
+fseek(file, 0L, SEEK_END);
+size_t file_size = ftell(file);
+fseek(file, 0L, SEEK_SET);
+
+wasm_byte_vec_t binary;
+wasm_byte_vec_new_uninitialized(&binary, file_size);
+
+if (fread(binary.data, file_size, 1, file) != 1) {
+  printf("> Error loading module!\n");
+  
+  return 1;
+}
+
+fclose(file);
+```
+{% endtab %}
+{% endtabs %}
+
+Let's assume we have the binary version of the module \(i.e the `.wasm` file\), here is how we would have loaded it:
+
+{% tabs %}
+{% tab title="Rust" %}
+```rust
+let wasm_bytes = include_bytes!("./path/to/module.wasm");
```

Comment:
I think that we should use `std::fs::read(wasm_path)` instead of `include_bytes!` (since the include bytes will include them on the binary)